pkg/setup/config.go

```diff
@@ -71,6 +73,19 @@ func InitAndValidateConfig(ctx context.Context, vConfig *config.VirtualClusterCo
 		return err
 	}
 
+	// set global owner for use in owner references
+	err = SetGlobalOwner(
+		ctx,
+		vConfig.ControlPlaneClient,
+		vConfig.WorkloadNamespace,
+		vConfig.WorkloadTargetNamespace,
+		vConfig.Experimental.SyncSettings.SetOwner,
+		vConfig.WorkloadService,
+	)
+	if err != nil {
+		return errors.Wrap(err, "finding vcluster pod owner")
```

Comment:
Given that it's just moved code, feel free to ignore it.

```suggestion
		return fmt.Errorf("finding vcluster pod owner: %w", err)
```